runtime/src/comm/gasnet/comm-gasnet.c

```diff
@@ -555,7 +555,7 @@ static void AM_shutdown(gasnet_token_t token) {
 static int bcast_seginfo_done = 0;
 static void AM_bcast_seginfo(gasnet_token_t token, void *buf, size_t nbytes) {
   assert(nbytes == sizeof(gasnet_seginfo_t)*gasnet_nodes());
-  chpl_memcpy(seginfo_table, buf, nbytes);
+  chpl_memmove(seginfo_table, buf, nbytes);
```

Comment:
This actually will overlap (perfectly) on node 0, but I believe that could be avoided by starting the for-stmt [here](https://github.com/chapel-lang/chapel/blob/45fff88adba59f298152fe44fac3a0628feca88c/runtime/src/comm/gasnet/comm-gasnet.c#L867) at `i=1` instead of `i=0`, because when `i==0` we're just sending to ourselves, it looks like to me.